servers/register_server_types.cpp

```diff
@@ -364,12 +363,14 @@ void unregister_server_types() {
 
 	ServersDebugger::deinitialize();
 	memdelete(shader_types);
+
 	if (GD_IS_CLASS_ENABLED(MovieWriterMJPEG)) {
 		memdelete(writer_mjpeg);
 	}
 	if (GD_IS_CLASS_ENABLED(MovieWriterPNGWAV)) {
 		memdelete(writer_pngwav);
 	}
+
```

Comment:
There's only whitespace changes in this file in the end so this can be reverted.

That should also prevent conflicts with #106013 (which could use an approval btw :wink: ).